modules/packages/Python.chpl

```diff
@@ -3580,6 +3581,23 @@ module Python {
     return v.str();
   }
 
+  // operator+(lhs: borrowed Value?, rhs: borrowed Value?): owned Value throws {
+  //   var lhs_ = (lhs:borrowed class);
+  //   var res = lhs_.call(PyObjectPtr, "__add__", rhs);
+  //   if res == Py_NotImplemented then
+  //     res = (rhs:borrowed class).call(PyObjectPtr, "__radd__", lhs);
+  //   return new Value(lhs_.interpreter, res, isOwned=true);
+  // }
+  // operator+=(ref lhs: owned Value?, rhs: borrowed Value?) throws do
+  //   lhs = (lhs:borrowed class) + rhs;
+
+
+  // operator==(lhs: borrowed Value, rhs: borrowed Value): bool throws do
+  //   return lhs.call(bool, "__eq__", rhs);
+  // operator!=(lhs: borrowed Value, rhs: borrowed Value): bool throws do
+  //   return lhs.call(bool, "__ne__", rhs);
+  // // ....
```

Comment:
This looks like an accidental add, but I'm also okay with leaving it in

Reply:
🤦. No this should not be added